tests/Facebook/InstantArticles/Transformer/wp-ia.xml

```diff
@@ -163,6 +163,15 @@
           <script async="" defer="defer" src="//platform.instagram.com/en_US/embeds.js"></script>
         </iframe>
       </figure>
+      <figure class="op-interactive">
+        <iframe class="no-margin">
+          <blockquote class="twitter-tweet" data-lang="en">
+      <p lang="en" dir="ltr">In an effort to establish stronger ties with the news industry, we're launching The Facebook Journalism Project: <a href="https://t.co/pSxENUU9aa">https://t.co/pSxENUU9aa</a></p>
```

Comment:
I had to change the indentation for the unit tests to pass, but I am not sure why this is happening.